pkgs/applications/virtualization/xen/generic/default.nix

```diff
@@ -691,10 +691,16 @@ stdenv.mkDerivation (finalAttrs: {
           # Finally, we write a notice explaining which vulnerabilities this Xen is NOT vulnerable to.
           # This will hopefully give users the peace of mind that their Xen is secure, without needing
           # to search the source code for the XSA patches.
-          + lib.strings.optionalString (writeAdvisoryDescription != [ ]) (
-            "\n\nThis Xen (${version}) has been patched against the following known security vulnerabilities:\n"
-            + lib.strings.removeSuffix "\n" (lib.strings.concatLines writeAdvisoryDescription)
-          );
+          + lib.strings.optionalString (writeAdvisoryDescription != [ ]) ''
+
+
+            <details>
+              <summary>This Xen (${version}) has been patched against the following known security vulnerabilities:</summary>
+
+            ${builtins.concatStringsSep "\n" writeAdvisoryDescription}
```

Comment:
Doesn’t using an HTML tag like this suppress Markdown syntax inside it? Are you sure this would result in paragraph breaks, etc.?

Probably it should be an actual list if we’re going to get this fancy, anyway. Perhaps even a definition list with the advisory number and description.

Reply:
<details>
  <summary><i>Doesn’t using an HTML tag like this suppress Markdown syntax inside it? Are you sure this would result in paragraph breaks, etc.?</i></summary>

I'm reasonably confident that it won't, as `nixos-search` seems to go line-by-line when replacing markdown for HTML. As long as we keep 

</details>

<details>
  <summary><i>Perhaps even a definition list with the advisory number and description.</i></summary>

Now *that* is just a marvelous idea. `<dt>` the current list topic and `<dl>` the `patch.meta.longDescription`. Should actually be reasonably easy to implement with the current functions.

</details>